docs/kyma-environment-broker/08-06-orchestration-status.md

```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
 A successful call returns a list of all orchestrations:
 
    ```json
-    [{
-      "state": "InProgress",
-      "description": "scheduled 5 operations",
+[
+  {
+      "orchestrationID": "07089b96-8e31-49a4-96d0-f8288253c804",
+      "state": "in progress",
+      "description": "Scheduled 2 operations",
+      "createdAt": "2020-10-12T19:27:22.281515Z",
+      "updatedAt": "2020-10-12T19:27:22.281515Z",
       "parameters": {
-        "targets": {
-          "include": [
-            {
-              "target": "all",
-            },
-          ],
-          "exclude": [
-            {
-              "runtimeID": "uuid",
-              "globalAccountId": "uuid",
-              "subAccountId": "uuid",
-              "region": "region",
-            },
-          ],
-        },
-        "strategy": ...
+          "targets": {
+              "include": [
+                  {
+                      "target": "all"
+                  }
+              ]
+          },
+          "strategy": {
+              "type": "parallel",
+              "schedule": "immediate",
+              "parallel": {
+                  "workers": 1
+              }
+          },
+          "dryRun": true
       }
-      "runtimeOperations": [
+  }
+]
+   ```
+
+## List the upgrade operations
```

Comment:
```suggestion
## List upgrade operations
```

Reply:
Shouldn't this heading match the bullet point from the overview section?

Reply:
we could go with `List upgrade operations scheduled by orchestration` but imo it's a bit too long like for a heading